patches/ungoogled-chromium/windows/windows-fix-command-ids.patch

```diff
@@ -0,0 +1,61 @@
+--- a/chrome/app/chrome_command_ids.h
++++ b/chrome/app/chrome_command_ids.h
+@@ -63,12 +63,6 @@
+ #define IDC_ALL_WINDOWS_FRONT           34048
+ #define IDC_NAME_WINDOW                 34049
+ 
+-// TODO(crbug.com/1052397): Revisit the macro expression once build flag switch of lacros-chrome is complete.
```

Comment:
Why do we need this patch? Are these defines set on Windows for some reason?

Reply:
This was mainly to get around build errors that occurred with this file:

```
[1575/46549] RC obj/chrome/app/chrome_dll_resources/chrome_dll.res
FAILED: obj/chrome/app/chrome_dll_resources/chrome_dll.res
C:/Python27/python.exe ../../build/toolchain/win/tool_wrapper.py rc-wrapper environment.x64 rc.exe /nologo "-imsvcC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29333\ATLMFC\include" "-imsvcC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29333\include" "-imsvcC:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\ucrt" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\shared" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\winrt" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\cppwinrt" -DUSE_AURA=1 "-DCR_CLANG_REVISION=\"llvmorg-12-init-16296-g5e476061-1\"" -D_HAS_NODISCARD -D_LIBCPP_ABI_UNSTABLE -D_LIBCPP_DISABLE_VISIBILITY_ANNOTATIONS -D_LIBCPP_ENABLE_NODISCARD -D_LIBCPP_NO_AUTO_LINK -D__STD_C -D_CRT_RAND_S -D_CRT_SECURE_NO_DEPRECATE -D_SCL_SECURE_NO_DEPRECATE -D_ATL_NO_OPENGL -D_WINDOWS -DCERT_CHAIN_PARA_HAS_EXTRA_FIELDS -DPSAPI_VERSION=2 -DWIN32 -D_SECURE_ATL -DWINAPI_FAMILY=WINAPI_FAMILY_DESKTOP_APP -DWIN32_LEAN_AND_MEAN -DNOMINMAX -D_UNICODE -DUNICODE -DNTDDI_VERSION=NTDDI_WIN10_VB -D_WIN32_WINNT=0x0A00 -DWINVER=0x0A00 -DNDEBUG -DNVALGRIND -DDYNAMIC_ANNOTATIONS_ENABLED=0 -I../.. -Igen /foobj/chrome/app/chrome_dll_resources/chrome_dll.res ../../chrome/app/chrome_dll.rc
../../chrome/app/chrome_command_ids.h(67) : warning RC4011: identifier truncated to 'BUILDFLAG_INTERNAL_IS_CHROMEOS_LACROSRNAL_, IS_CHROMEOS_LACROS)pwinrt\build/buildflag.h'

../../chrome/app/chrome_command_ids.h(67) : fatal error RC1012: mismatched parenthesis : missing '
```

Reply:
@tangalbert919 weird, my builds (both 89 and 90) work just fine without the patch. I'm also building on windows and using vs 2019 community, btw.